### DISCUSSION TITLE
Rename Singles Option

### DISCUSSION THREAD

**Discussion by angrybrad** [craftcms/cms#8141] [Ideas]
> Created by: Julian Rutherford (justjoolz@gmail.com) on 2015/04/19 11:42:02 +0000
> Votes at time of UserVoice import: 10

-----------------------------------

The ability to add headers in the entries section and rearrange the order of channels etc. in 2.5 is really great.

Continuing in this customizing direction the ability to change the title of 'singles' to something else from the gear panel would be really great.

**Comment by angrybrad** [comment]
> Posted by Visiluna (pjones@visiluna.com) on 2015/06/16 22:04:13 +0000
-----------------------------------

As a developer, I understand the meaning of &quot;Singles&quot; in the context of Craft, but it is confusing to many of my clients. I&#39;ve had to explain to more than one that it has nothing to do with unmarried people. (I&#39;m not joking).

Since I have total control over the naming of each section, and now the headings under which each section appears in the CP, &quot;Singles&quot; is the last bit that I can&#39;t currently customize.

I would be okay with either a CP field or a config variable.

**Comment by angrybrad** [comment]
> Posted by Collin Reisdorf (collin@paperclipped.com) on 2016/08/04 18:39:06 +0000
-----------------------------------

Even a clear config.php way to rename it would be great. Thanks!

**Comment by matthewstick** [comment]
Will this be done?  Has anyone found a plugin?

**Comment by matthewstick** [comment]
I figured out a work around.  You need to set up a translation file.

So in my case I made an "en" folder for English, and then re-mapped it.

This linked helped:
https://cl.ly/6e5132433d70

**Comment by daveyjones** [comment]
@angrybrad I have the same issue. I always tell clients to read "Singles" as "Pages", because that's almost always what singles represent in my projects.

@matthewstick Great workaround!
